### PR TITLE
Convert our leavers document to a runbook

### PR DESCRIPTION
**Pull request by pwyborn**
Why - ticket  https://app.zenhub.com/workspaces/cloud-platform-team-5ccb0b8a81f66118c983c189/issues/ministryofjustice/cloud-platform/1041 Convert our leavers document to a runbook

### CODE REVIEW

File: runbooks/source/leavers-guide.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+## Revoking Access
+
+### Digital Services
+
+The central Digital Services team will be able to remove and revoke access from a number of different accounts outside of the technical team.
+
+Contact the [Service Desk](mailto:servicedesk%40digital.justice.gov.uk)  (or slack channel #digitalservicedesk), who will carry out the following actions [for further information](https://dsdmoj.atlassian.net/wiki/spaces/DSD/pages/44957720/Leavers):
+
+* Confirm to the team member & line manager what equipment & asset numbers are expected to be returned
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: I don't think we should duplicate the information in the Digital Service Desk's Confluence page. Their leaver's tasks will change, and we shouldn't have to update our runbook when that happens.

Personally, I'd suggest replacing everything from "Contact the Service Desk..." to "Reset locker code..." with something like this:

> Contact the [Service Desk] (or slack...). They will go through their own leaver's process, as documented [here].

File: runbooks/source/leavers-guide.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+
+* [Pagerduty](https://moj-digital-tools.pagerduty.com/users)
+
+* [Heroku](https://devcenter.heroku.com/articles/org-users-access)
+
+* [Sentry](https://sentry.service.dsd.io/organizations/mojds/members/)
+
+* [CI](https://ci.service.dsd.io/configureSecurity/)
+
+* [Rattic](https://rattic.service.dsd.io/staff/). In process of being replaced by LastPass (please see Rattic link)
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: I think Rattic has already been replaced with LastPass, so I'd replace this item with a link to our LastPass account (or instructions on how to get a LastPass admin (Steve?) to remove the user).

File: runbooks/source/leavers-guide.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+
+#### 3/ Contact the VPN provider to remove VPN access (whilst it is Digital Services responsibility I prefer to email the vpn provider myself)
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: Who is the VPN provider, and how do we email them?

File: runbooks/source/leavers-guide.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+
+#### 4/ Remove them from the support rota
+
+#### 5/ Remove them from platforms@digital.justice.gov.uk google group
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: How?

File: runbooks/source/leavers-guide.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+
+#### 3/ Contact the VPN provider to remove VPN access (whilst it is Digital Services responsibility I prefer to email the vpn provider myself)
+
+#### 4/ Remove them from the support rota
+
+#### 5/ Remove them from platforms@digital.justice.gov.uk google group 
+
+Reference [platforms@digital.justice.gov.uk](mailto:platforms@digital.justice.gov.uk)
+
+#### 6/ Remove them from MoJ organisation membership on GitHub (Whilst it is Digital Services responsibility - I prefer to do this to enable you carry out the fab salt.ssh_key actions following at 7/).
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: 6 & 7 seem to be specific to Template Deploy. I'd suggest highlighting this with a section heading, something like: 

> **Template Deploy**
> Remove this section once it is no longer needed
> ...

File: runbooks/source/leavers-guide.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,415 @@
...
+
+* [Sentry](https://sentry.service.dsd.io/organizations/mojds/members/)
+
+* [CI](https://ci.service.dsd.io/configureSecurity/)
+
+* [Rattic](https://rattic.service.dsd.io/staff/). In process of being replaced by LastPass (please see Rattic link)
+
+#### 3/ Contact the VPN provider to remove VPN access (whilst it is Digital Services responsibility I prefer to email the vpn provider myself)
+
+#### 4/ Remove them from the support rota
```
Comment (digitalronin) [inline runbooks/source/leavers-guide.html.md.erb]: How?

File: runbooks/source/leavers-guide.html.md.erb (hunk 2 above)
Comment (sid234) [reply runbooks/source/leavers-guide.html.md.erb]: the last pass admin link is present in the rattic link which can be accessed internally. this is an extra step and can be accessed from the inside. cause the runbook will be published public, the question is do we publish the details of the last pass admin link.

File: runbooks/source/leavers-guide.html.md.erb (hunk 3 above)
Comment (sid234) [reply runbooks/source/leavers-guide.html.md.erb]: could we get the details of the VPN provider from digital service desk? mentioning the details of vpn is exposing it on the public. maybe we could have a runbook accessed internally?

File: runbooks/source/leavers-guide.html.md.erb (hunk 2 above)
Comment (pwyborn) [reply runbooks/source/leavers-guide.html.md.erb]: I have put the email contact details in a shared google drive document now

File: runbooks/source/leavers-guide.html.md.erb (hunk 3 above)
Comment (pwyborn) [reply runbooks/source/leavers-guide.html.md.erb]: I have put the email contact details in a shared google drive document now

**Comment by pwyborn** [comment]
Hi @digitalronin / @sid234 Please can you re-review my changes (with reference to previous review conversations) Thank you

**Pull request merged**